### ISSUE TITLE
Timecodes for "ML Zoomcamp 6.7 - Gradient Boosting and XGBoost"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#169]
Youtube video: https://www.youtube.com/watch?v=xFarGClszEM

**Comment by dimzachar** [comment]
0:00:00 - Lesson 7: Gradient Boosting explained.
0:02:06 - Sequential training of gradient boosted trees.
0:04:13 - Training XGBoost model with specified parameters.
0:06:05 - Setting parameters for binary classification model.
0:07:39 - Creating and monitoring XGBoost models' performance.
0:09:24 - Training and evaluating model with AUC.
0:11:14 - Expert: Plotting and printing model performance.
0:12:53 - Parsing captured output into train/validation data.
0:14:38 - Extracting iteration and performance data for analysis.
0:16:30 - Plotting model performance and overfitting detection.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
